ansible_collections/arista/avd/docs/pyavd.md

```diff
@@ -68,7 +68,7 @@ Requirements (automatically installed with above command):
 
 ```ini
 jinja2>=3.0
-jsonschema>=4.5.1,<4.18
+jsonschema>=4.5.1
```

Comment:
does not look align with release notes and 4.10.3

Reply:
fixed in 66384c3